### DISCUSSION TITLE
Obsolete "{airHome}/frameworks/" reference in air-config.xml

### DISCUSSION THREAD

**Discussion by 2jfw** [airsdk/Adobe-Runtime-Support#1763] [Q&A] [answered]
In the AIR SDK in folder `frameworks` in file `air-config.xml` there are two (possibly obsolete?) path references to `{airHome}/frameworks/`

**{airHome}/frameworks/libs/air/airglobal.swc**:

`	  <!-- List of SWC files or directories to compile against but to omit from -->`
`      <!-- linking.                                                             -->`
`      <external-library-path>`
`       <path-element>{airHome}/frameworks/libs/air/airglobal.swc</path-element>`
`      </external-library-path>`

and

**{airHome}/frameworks/libs/air**:

` <!-- List of SWC files or directories that contain SWC files. -->`
`      <library-path>`
`         <path-element>libs</path-element>`
`         <path-element>libs/mx</path-element>`
`         <path-element>libs/air</path-element>`
`         <path-element>{airHome}/frameworks/libs/air</path-element>`
`         <path-element>locale/{locale}</path-element>`
`      </library-path>`

Locally it does not affect compiling; however, our Buildserver (Jenkins) fails when `{airHome}/frameworks/` is not removed from those two elements (whereas removing the entire line `         <path-element>{airHome}/frameworks/libs/air</path-element>` for the second occurrence, since it is duplicating the above one [libs/air])

@ajwfrost: Is there any reason to have the `{airHome}/frameworks/`-addition on those two items?

On both, my local machine and the buildserver there is no `enviroment variable` with the name `airHome` and I would think the `{airHome}` is referring to that (?)...

**Comment by 2jfw** [comment, marked as answer]
As a short update on this: the file is actually part of the Flex SDK/Runtime and **not** part of the AIR SDK so this matter itself is "obsolete"

**Reply by ajwfrost** [reply]
Looks like I missed seeing this originally .. I believe that those values within { } are meant to be updated upon installation by the Flex SDK Installer application (which I'm not sure is available any more...). The versions of those files that we ship as part of the AIR SDK haven't got the `{airHome}` value in now, but I've just noticed the use of `locale/{locale}` still. I think this means it will just be ignored, and probably people are passing in the locale via the build options, but if anyone finds this is a problem we'll have to investigate further. It's only used by Flex apps though afaik so probably our versions of the files would not be used anyway!

thanks

**Reply by 2jfw** [reply]
No problem and thanks for the details :)
